test/intl402/Temporal/TimeZone/etc-timezone.js

```diff
@@ -10,27 +10,34 @@ features: [Temporal]
 // "Etc/GMT-0" through "Etc/GMT-14" are OK
 
 assert.sameValue(
-  Temporal.TimeZone.from("Etc/GMT-0").toString(),
+  (new Temporal.TimeZone("Etc/GMT-0")).toString(),
```

Comment:
What motivates `new Temporal.TimeZone(…)` vs. `Temporal.TimeZone.from(…)`?

Reply:
That was my suggestion, I'd prefer to use the constructor in the tests unless we are specifically testing functionality of from(), so that the results aren't affected if there is some other bug in from().